# 数据存放路径
data_path: /media/lxhong/Datasets

# 局部地图
local_frame_num: 20

# 各配置选项对应参数
## 滤波相关参数
global_map:
    leaf_size: [0.5, 0.5, 0.5]
local_map:
    leaf_size: [0.5, 0.5, 0.5]
frame:
    leaf_size: [0.5, 0.5, 0.5]
